src/webapp/components/survey-list/SurveyListFilters.tsx

```diff
@@ -0,0 +1,78 @@
+import i18n from "@eyeseetea/feedback-component/locales";
+import { FormControl, InputLabel, MenuItem, Select } from "@material-ui/core";
+import { Dispatch, SetStateAction } from "react";
+import styled from "styled-components";
+import { SURVEY_STATUSES, SURVEY_TYPES } from "../../../domain/entities/Survey";
+
+interface SurveyListFilterProps {
+    status: SURVEY_STATUSES | undefined;
+    setStatus: Dispatch<SetStateAction<SURVEY_STATUSES | undefined>>;
+    surveyType: SURVEY_TYPES | undefined;
+    setSurveyType: Dispatch<SetStateAction<SURVEY_TYPES | undefined>>;
+}
+export const SurveyListFilters: React.FC<SurveyListFilterProps> = ({
+    status,
+    setStatus,
+    surveyType,
+    setSurveyType,
+}) => {
+    return (
+        <FilterContainer>
+            <FormControl fullWidth>
+                <InputLabel id="status-label">Filter by Status</InputLabel>
+                <Select
+                    labelId="status-label"
+                    value={status}
+                    onChange={e => {
+                        if (e.target.value === "NONE") setStatus(undefined);
+                        else setStatus(e.target.value as SURVEY_STATUSES);
+                    }}
+                    label="STATUS"
+                >
+                    <MenuItem key="COMPLETED" value="COMPLETED">
+                        {i18n.t("COMPLETED")}
+                    </MenuItem>
+                    <MenuItem key="ACTIVE" value="ACTIVE">
+                        {i18n.t("ACTIVE")}
+                    </MenuItem>
+                    <MenuItem key="FUTURE" value="FUTURE">
+                        {i18n.t("FUTURE")}
+                    </MenuItem>
+                    <MenuItem key="NONE-Status" value="NONE">
+                        {i18n.t("NONE")}
+                    </MenuItem>
+                </Select>
+            </FormControl>
+
+            <FormControl fullWidth>
+                <InputLabel id="status-label">Filter by Survey Type</InputLabel>
```

Comment:
i18n here

Reply:
done in https://github.com/EyeSeeTea/amr-surveys/pull/11